### DISCUSSION TITLE
Does the JS code loaded through scripts gets compiled for cross-browser compatibility?

### DISCUSSION THREAD

**Discussion by ghost** [webdiscus/pug-plugin#90] [Q&A]
Just the title, if not can I use Babel to compile the code? How?

Thanks in advance.

**Comment by webdiscus** [comment]
Hello @racc-ode,

all source scripts `*.js`, `*.ts`, etc required in the Pug template will be compiled via Webpack for a browser.
If you want have the JS code compatible with a specification, e.g. `IE`, `ES5`, etc., you can use the [babel-loader](https://webpack.js.org/loaders/babel-loader/). This loader will make your code compatible with anything you want.
